src/code/crosslinks.ts

```diff
@@ -333,13 +343,21 @@ export function checkAllLinks() {
   }
 
   for (const l of operation.links) {
+    // Highlight if crosses any other Wasabee links
     if (testSelfBlock(l, operation)) {
-      const blocked = L.geodesicPolyline(
-        l.getLatLngs(operation),
-        window.plugin.wasabee.skin.selfBlockStyle
-      );
-      blocked.options.interactive = false;
-      blocked.addTo(window.plugin.wasabee.crossLinkLayers);
+      addToCrossLinkLayers(l,operation,window.plugin.wasabee.skin.selfBlockStyle);
+    }
+    else
+    {
+      const rlinkGenerator = realLinks();
+      //  Highlight if crosses any real links without markers
+      for (const rlink of rlinkGenerator) {
+        if (testPolyLine(l, rlink, operation))
+        {
+          addToCrossLinkLayers(l,operation,window.plugin.wasabee.skin.selfBlockStyle);
```

Comment:
you can introduce a specific style if needed (would need to edit static.ts and skin.ts)